vanilla-todo/src/services/TodoService.js

```diff
@@ -0,0 +1,56 @@
+class TodoService {
+  #todos = [];
+
+  get todos() {
+    return this.#todos.map((todo) => Object.freeze(todo));
+  }
+  findTodo(todoId) {
+    const todoIndex = this.#todos.findIndex((value) => value.id === todoId);
+    const todo = this.#todos.find((value) => value.id === todoId);
+    return { todo, index: this.#todos[todoIndex] };
+  }
```

Comment:
```suggestion
  findTodo(todoId) {
    const todoIndex = this.#todos.findIndex((value) => value.id === todoId);
    return { todo: this.#todos[todoIndex], index: todoIndex };
  }
```
이렇게 표현해야 정상적으로 동작할 것 같네요!